.dependabot/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+update_configs:
+- package_manager: "java:maven"
+  directory: "/"
+  update_schedule: "weekly"
+  default_reviewers:
+    - "aheritier"
+    - "kwhetstone"
+    - "Evildethow"
+  ignored_updates:
+    - match:
+        dependency_name: "org.jenkins-ci.main:jenkins-core"
+    - match:
+        dependency_name: "org.jenkins-ci.plugins*"
```

Comment:
Note that it will filter out Plugin POM. If you do not depend on too many plugins, maybe better to unfilter it and ignore plugins explicitly

Reply:
good point